introduction.md

```diff
@@ -6,33 +6,45 @@ layout: documentation
 
 # About openHAB
 
-openHAB is **software** for integrating different home automation systems and technologies into one single solution that allows over-arching automation rules, and that offers uniform user interfaces.
+openHAB **software** integrates different home automation systems, devices and technologies into a single solution.
+It provides uniform user interfaces, and a common approach to automation rules across the entire system, regardless of the number of manufacturers and sub-systems involved.
 
-But openHAB is also a **community** of users, contributors and maintainers.
+## Getting Started
 
-## openHAB Structure
-
-openHAB 2 is an open-source solution based on the [Eclipse SmartHome](https://eclipse.org/smarthome/) framework. It is written in Java and uses [Apache Karaf](http://karaf.apache.org/) together with [Eclipse Equinox](https://www.eclipse.org/equinox/) as an OSGi runtime and bundles this with [Jetty](https://www.eclipse.org/jetty/) as an HTTP server.
+We highly recommend that you read the next article titled [Concepts]({{base}}/concepts/index.html).
```

Comment:
While we are talking about terminology: As the Concepts part is an area with more than one article, isn't it a chapter?

Reply:
Yes - I agree and will change this with my next commit.

I nominate the following convention for all documentation in opehHAB...

* Documentation <- the overall documentation package
* Chapter <- each first-level page of the overall documentation
* Article <- each second-level page of a Chapter
* Section <- each first-level part of a page, whether that page is a Chapter or an Article
* Sub-section <- each second-level part of a Section of a page, whether that page is a Chapter or an article
* Paragraph <- just what it says
* Sentence <- just what it says

If you agree, probably best to pull this out of a note buried in a PR and stick it somewhere else - no?